clients/fides-js/src/lib/consent-types.ts

```diff
@@ -138,16 +140,20 @@ export enum ButtonType {
   TERTIARY = "tertiary",
 }
 
-export type PrivacyPreferencesCreateWithCode = {
-  // TODO: update this schema
+export enum ConsentMethod {
+  button = "button",
+  gpc = "gpc",
+  individual_notice = "api",
```

Comment:
Why the rename?

Reply:
I was using the incorrect type from the BE before, so I copied over the correct one here, using the same naming.